app/entry.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+require('./scss/reset.scss');
+require('./scss/main.scss');
```

Comment:
These two lines are breaking your compile and causing your tests to fail.  You have partially refactored this but it isn't quite done.